bank_failures/python_vis/visuals.py

```diff
@@ -2,6 +2,7 @@
 import seaborn as sns
 import matplotlib.pyplot as plt
 
+print("This is my code change.")
```

Comment:
Comment: Hey, this isn't the right code format, could you please use our standard format